test/controllers/ChooseAnOptionControllerSpec.scala

```diff
@@ -629,4 +629,13 @@ class ChooseAnOptionControllerSpec extends UnitSpec with I18nHelper with RasTest
       doc(result).getElementById("upload-link").attr("data-journey-click") shouldBe "link - click:You have not uploaded a file:Upload a file"
     }
   }
+
+  "fomattedExpiryDate method" should {
+    "return correctly formatted date and time" in {
+      val date = DateTime.now
```

Comment:
I would actually set this date to a fixed value rather than DateTime.now so we don't have any variance in our tests. And for the assertion, I would actually write the string as "May 1st 2020" or whatever the format it's meant to be because by reading the test I still don't actually know what the output is.

Reply:
That's a good call, just updated this now